components/dropdown/DropdownMenuItem.vue

```diff
@@ -18,29 +18,31 @@ const component = computed(() =>
 </script>
 <template>
   <MenuItem v-slot="{ active, close }" as="template">
-    <component
-      :is="component"
-      :to="to"
-      :href="href"
-      :class="{
-        'bg-label-separator': active,
-      }"
-      class="type-subtitle-2 flex w-full items-center justify-between rounded-lg px-3 py-2 text-left"
-    >
-      <slot :active="active">
-        <div class="flex-1" @click="to && close()">
-          <span class="flex items-center gap-2">
-            <NuxtIcon v-if="icon" :name="icon" />
-            {{ title }}
-          </span>
-          <span class="type-subtitle-3 text-label-3">
-            {{ secondaryTitle }}
-          </span>
-        </div>
-        <div>
-          <slot name="right" />
-        </div>
-      </slot>
-    </component>
+    <slot name="item" :active="active">
+      <component
+        :is="component"
+        :to="to"
+        :href="href"
+        :class="{
+          'bg-label-separator': active,
+        }"
+        class="type-subtitle-2 flex w-full items-center justify-between rounded-lg px-3 py-2 text-left"
```

Comment:
can you add whitespace-normal. That prevents a blowout if you have many content languages

Reply:
@kkuepper Does this look good? I added `truncate` to the `DropdownMenuItem` and added a max-width to the `DropdownMenu`.

![CleanShot 2024-06-26 at 18 39 11@2x](https://github.com/bcc-code/bmm-web/assets/18753964/6ef5007c-9d8f-40a5-b8d3-c4df26c7e269)

Reply:
That works